Address FIXME about term for UEFI/BIOS/Firmware (#85)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

{{layered}} depicts an example of a device that includes (A) a BIOS stored
in read-only memory in this example, (B) an updatable bootloader, and (C)
an operating system kernel.

{:layered: artwork-align="center"}
~~~~ LAYERED
{::include layered-attester.txt}
~~~~
{:layered #layered title="Layered Attester"}

Attesting Environment A, the read-only BIOS in this example,
has to ensure the integrity of the bootloader (Target Environment B).
There are
potentially multiple kernels to boot, and the decision is up to the bootloader.
Only a bootloader with intact integrity will make an appropriate decision. Therefore, these Claims have to be measured securely.
At this stage of the boot-cycle of the
device, the Claims collected typically cannot be composed into Evidence.

After the boot sequence is started, the BIOS conducts the
most important and defining feature of layered attestation, which is that
the successfully measured Target Environment B
now becomes (or contains) an Attesting Environment for the next layer.
This procedure in Layered Attestation is sometimes called "staging".
It is important to note that the new Attesting Environment B cannot alter

> any Claims about its own Target Environment B, since those Claims were signed

Reviewer: When we say "signed" here are we implying a signing operation done by A in the cryptographic sense, i.e. using A's private key?  If so, I'm not sure this is the necessary precondition.  Measurements might just be "sealed" in the RoT at a certain boot stage for later use by the lead attester.

Author: Good point, updated.

an operating system kernel.

{:layered: artwork-align="center"}
~~~~ LAYERED
{::include layered-attester.txt}
~~~~
{:layered #layered title="Layered Attester"}

Attesting Environment A, the read-only BIOS in this example,
has to ensure the integrity of the bootloader (Target Environment B).
There are
potentially multiple kernels to boot, and the decision is up to the bootloader.
Only a bootloader with intact integrity will make an appropriate decision. Therefore, these Claims have to be measured securely.
At this stage of the boot-cycle of the
device, the Claims collected typically cannot be composed into Evidence.

After the boot sequence is started, the BIOS conducts the
most important and defining feature of layered attestation, which is that
the successfully measured Target Environment B
now becomes (or contains) an Attesting Environment for the next layer.
This procedure in Layered Attestation is sometimes called "staging".
It is important that the new Attesting Environment B not be
able to alter any Claims about its own Target Environment B. 
This can be ensured having those Claims be either signed by Attesting

> Environment A or stored in a shielded and secure manner by Attesting

Author: ```suggestion
Environment A or stored in an untamperable manner by Attesting
```
